review/tests.py

```diff
@@ -2,52 +2,48 @@
-RATING = 5
-DAY_CARE_ID = 1
-DOG_OWNER_ID = 1
 
-
-@pytest.mark.django_db
-@pytest.fixture
-def test_review():
-    return Review.create(review=REVIEW, rating=RATING, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
-
-
-@pytest.mark.django_db
+@pytest.mark.django_db()
 class TestReviewModel:
-    def test_persist_review(self, test_review):
-        assert test_review.review == REVIEW
-        assert test_review in Review.objects.all()
+    def test_persist_review(self, review, review_data):
+        assert review.review == pytest.REVIEW
+        assert review in Review.objects.all()
 
-    def test_del_review(self, test_review):
-        test_review.delete()
-        assert test_review not in Review.objects.all()
-
-    def test_rating_limit_min(self):
-        with pytest.raises(ValidationError):
-            Review.create(review=REVIEW, rating=-1, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
+    def test_del_review(self, review):
+        review.delete()
+        assert review not in Review.objects.all()
 
-    def test_rating_limit_max(self):
+    @pytest.mark.parametrize("rating", [-1, 6])
+    def test_rating_limit_min(self, rating, review_data):
         with pytest.raises(ValidationError):
-            Review.create(review=REVIEW, rating=6, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
+            Review.create(review=pytest.REVIEW, rating=rating, daycare_id=pytest.DAY_CARE_ID,
+                          dogowner_id=pytest.DOG_OWNER_ID)
 
-    def test_delete_related_daycare(self, test_review):
-        daycare = DayCare.objects.get(pk=DAY_CARE_ID)
+    def test_delete_related_daycare(self, review, review_data):
+        daycare = DayCare.objects.get(pk=pytest.DAY_CARE_ID)
 
         assert daycare in DayCare.objects.all()
-        assert test_review in Review.objects.all()
+        assert review in Review.objects.all()
 
         daycare.delete()
         assert daycare not in DayCare.objects.all()
-        assert test_review not in Review.objects.all()
-
-    def test_delete_related_dogowner(self, test_review):
-        dogowner = DogOwner.objects.get(id=DOG_OWNER_ID)
+        assert review not in Review.objects.all()
 
+    def test_delete_related_dogowner(self, dogowner, review, review_data):
         assert dogowner in DogOwner.objects.all()
-        assert test_review in Review.objects.all()
+        assert review in Review.objects.all()
 
         dogowner.delete()
         assert dogowner not in DogOwner.objects.all()
-        assert test_review in Review.objects.all()
+        assert review in Review.objects.all()
+
+    @pytest.mark.parametrize("number_of_reviews", [0, 1, 3])
+    def test_get_review_by_daycare_id(self, number_of_reviews, dogowner, daycare):
+        review_list = [Review.create(review='review', rating=1, daycare_id=daycare.id, dogowner_id=dogowner.id)
+                       for _ in range(number_of_reviews)]
+        reviews_response = Review.get_review_by_daycare_id(daycare.id)
```

Comment:
It's a nitpick, but still important to understand:
to improve test readability, the spaces should divide the arrange, act, and assert parts of the test. 
the `review_list` is the arrange, the `reviews_response` is the act (it uses the method this test is made for) and the rest is the assert. 

p.s: long parts can use spaces as well. but this is irrelevant to this test